backend/src/org/commcare/util/SessionNavigationResponder.java

```diff
@@ -0,0 +1,22 @@
+package org.commcare.util;
```

Comment:
Agreed, but I put these files here because this is where CommCareSession.java is also. Happy to move them all into a session-related package.